content/customize-gnome.md

```diff
@@ -18,117 +18,210 @@ section: software
 tableOfContents: true
 ---
 
-The GNOME desktop environment is highly customizable.  With a few extensions, the desktop can look and feel like many other operating system's desktops.  Launcher bars can be added, window lists can be used, and the overall look and feel of the desktop can be changed.
+The GNOME desktop environment is highly customizable. Visit [extensions.gnome.org](https://extensions.gnome.org) to install, remove, and configure currently installed extensions.  A web browser plugin is needed to interact with the site, and it will prompt for installation if needed.  Remember to refresh the page after the plugin is installed.
 
-To install a customization, simply head to [extensions.gnome.org](https://extensions.gnome.org) to install, remove, and configure currently installed customizations.  A web browser plugin is needed to interact with the site, and it will prompt for installation if needed.  Remember to refresh the page after the plugin is installed.
-
-We will share some of the more popular extensions, but there are many more to choose from!  If one is found and it isn't currently working, or breaks with an update, then try a different extension to see if it works better.
+**NOTE:** There are many extensions to choose from! If one isn't currently working, or breaks with an update, then try a different extension to see if it works better.
 
 ---
 
-## User Themes
+## Included Extensions
 
-[User Themes](https://extensions.gnome.org/extension/19/user-themes/)
+<details>
+  <summary>Click to expand</summary>
 
-This extension allows for the desktop to be themed in further ways than default.  Many themes and extensions can be installed from [opendesktop.org](https://www.opendesktop.org/s/Gnome), among others.  This extension is installed by default in Pop!_OS, and can be added to Ubuntu 18.04.
+Pop!\_OS includes the GNOME ```Extensions``` app by default. Which lists both system-installed, and user-installed extensions.
```

Comment:
> Which lists both system-installed, and user-installed extensions.

This is not a complete sentence. "Which" starts a dependent clause. Also, there should be no comma before "and" because there are only two items in the list.

Also, inline code highlighting should only use a single backtick on either side of the word. Three backticks should only be used for a code block. See https://www.markdownguide.org/basic-syntax/#code.

I would suggest:

> Pop!\_OS includes the GNOME `Extensions` app by default, which lists both system-installed and user-installed extensions.

Reply:
Fixed.